examples/resources/genesyscloud_journey_outcome_predictor/apis.md

```diff
@@ -0,0 +1,7 @@
+* [GET /api/v2/journey/outcomes/predictors](https://apicentral.genesys.cloud/api-explorer#get-api-v2-journey-outcomes-predictors)
+* [POST /api/v2/journey/outcomes/predictors](https://apicentral.genesys.cloud/api-explorer#post-api-v2-journey-outcomes-predictors)
+* [GET /api/v2/journey/outcomes/predictors/{predictorId}](https://apicentral.genesys.cloud/api-explorer#get-api-v2-journey-outcomes-predictors--predictorId-)
+* [DELETE /api/v2/journey/outcomes/predictors/{predictorId}](https://apicentral.genesys.cloud/api-explorer#delete-api-v2-journey-outcomes-predictors--predictorId-)
+
```

Comment:
Not a big deal but there's some whitespace in this file, it's creating a bit of a gap in the documentation